controllers/members.js

```diff
@@ -83,11 +84,22 @@ const archiveMembers = async (req, res) => {
   try {
     const { username } = req.params;
     const user = await dataAccess.retrieveUsers({ username });
+    const superUserId = req.userData.id;
+    const body = req.body;
+    const meta = {
+      userId: user?.user?.id,
+      superUserId: superUserId,
+      username: username,
```

Comment:
what is meta here?? when I read this variable name first thought I get is of meta tag..is it something related to that...??

Reply:
Since in our `logs.js`, we have a function to `addLogs`, which takes three param - `type`, `meta` and `body`, 
so passing the `meta` here into that function as a param and keeping the object name the same for reference i.e. `meta`